src/main/resources/static/apps/admin/modules/core/userProfile.html

```diff
@@ -52,6 +53,35 @@
                         </div>
                     </div>
                 </form>
+
+                <form name="pwdForm" class="col-sm-8 col-sm-offset-2">
+                    <h2 class="with-border">{{'admin.modules.core.userProfile.titleChangePassword'|translate}}</h2>
+                    <div class="row">
+                        <div class="col-xs-7">
+                            <div class="form-group" ng-class="{'has-error': pwdForm.oldPassword.$dirty && pwdForm.oldPassword.$touched && pwdForm.oldPassword.$invalid}">
```

Comment:
I wrote a component for form-group, you can use it instead. For example:
```html
<ui-input name="realName"
                label="admin.modules.core.userProfile.labelRealName"
                ng-model="viewModel.realName"></ui-input>
```

Reply:
:clap: Cool!